docs/DeveloperGuide.md

```diff
@@ -320,26 +321,26 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
-| `* * *`  | CCA leader with many tasks to be done in the CCA | be able to add tasks into the application | better manage the tasks |
-| `* * *`  | CCA leader with tasks that are constantly changing in details | be able to edit the details of the tasks accordingly | keep the task details updated |
+| `* * *`  | CCA leader with many tasks to be done | be able to add the tasks of our CCA | better manage the tasks |
+| `* * *`  | CCA leader whose tasks' details are needed to be updated constantly | be able to edit the details of the tasks accordingly | keep the tasks' details updated |
 | `* * *`  | CCA leader with tasks that are no longer relevant | be able to delete tasks that are either completed or unnecessary | keep the list organized and clear |
 | `* * *`  | CCA leader with projects and tasks piling up | be able to view a list of tasks that is to be completed by members of the CCA | be updated with what needs to be done |
 | `* * *`  | CCA leader who has members leaving the CCA | be able to remove details of the user from that application | better manage my members |
 | `* * *`  | CCA leader whose members’ details have changed | be able to edit the details of the user in the application | keep my contacts up-to-date |
 | `* * *` | CCA leader with many deadlines to meet | be able to set deadlines for specific tasks | keep track of the tasks' deadlines |
-| `* * *` | CCA leader with tasks to distribute | be able to assign tasks to students in the application | track the distribution of tasks |
+| `* * *` | CCA leader with tasks to be distributed | be able to assign tasks to students | track the distribution of tasks to certain members|
 | `* * `  | CCA leader with tasks that are being completed every other week | be able to mark tasks as ‘completed’  | track which tasks are completed |
 | `* * `  | CCA leader | be able to change the assignee of a task | easily update the member assigned to the task |
-| `* * `  | CCA leader | be able to assign a priority to a task | see which tasks need to be completed first |
-| `* * `  | CCA leader who has many tasks to supervise | be able to search for tasks which are of a certain priority | better manage tasks of different priorities |
-| `* * `  | CCA leader whose members have different roles | be able to assign roles to the members in the application | know who has a certain role |
-| `* * `  | CCA leader whose members have different roles | be able to edit the roles to the members in the application  | update the member roles when there is a change their roles |
-| `* `  | CCA leader who is very forgetful | be notified if I have accidentally added duplicate items/tasks | prevent repeated tasks from being added |
-| `* `  | CCA leader who has a lot of tasks to manage | search for tasks which contain a particular word  | find tasks more efficiently |
+| `* * `  | CCA leader | be able to assign a priority to a task | take note of the tasks are more important and need to be completed first |
```

Comment:
"take note of the tasks that are more important"

Reply:
okay